locales/cy/translations.json

```diff
@@ -1,5 +1,6 @@
 {
     "buttons" : {
+        "continue" : "WELCH - Continue",
```

Comment:
Typo in language name WELCH should be WELSH

Reply:
typo fixed